atom/browser/api/atom_api_top_level_window.cc

```diff
@@ -691,9 +691,16 @@ void TopLevelWindow::SetOverlayIcon(const gfx::Image& overlay,
   window_->SetOverlayIcon(overlay, description);
 }
 
+#if defined(OS_MACOSX)
```

Comment:
Can we drop the `#if defined(X)` everywhere, it adds a bit of complexity to the API that is not required.

We can pass `mate::Arguments* args` here and then rip an optional boolean out on the macOS implementation to avoid the defined checks everywhere

Reply:
fixed! Let me know if I'm the right path now